external/builder/babel-plugin-pdfjs-preprocessor.mjs

```diff
@@ -173,6 +173,36 @@ function babelPluginPDFJSPreprocessor(babel, ctx) {
           path.replaceWith(t.importExpression(source));
         }
       },
+      NewExpression(path) {
+        const { node } = path;
+
+        if (
+          t.isIdentifier(node.callee, { name: "URL" }) &&
+          node.arguments.length === 2
+        ) {
+          const [arg1, arg2] = node.arguments;
+
+          if (
+            arg1.type === "StringLiteral" &&
+            arg1.value.endsWith(".wasm") &&
+            arg2.type === "MemberExpression"
```

Comment:
The Babel plugin changes look good. Out of curiosity, is there any case in the codebase in which this MemberExpression check actually makes a difference?

Reply:
Not right now, as far as I can tell; my idea was just to reduce the risk of a future bug by being more specific.

Ideally we should check that the second argument is explicitly `import.meta.url`, however that seemed (with my limited Babel experience) to be fairly unwieldy so I figured that a simpler check might be good enough!?

Reply:
Yeah, I just opened https://github.com/babel/babel/issues/17225 to make it an easier check, but for now it would indeed be annoying to do it.